docs/reference/schemas/config/metadata.md

```diff
@@ -0,0 +1,80 @@
+---
+description: JSON schema reference for metadata in a Desired State Configuration document.
+ms.date:     01/17/2024
+ms.topic:    reference
+title:       DSC Configuration document metadata schema
+---
+
+# DSC Configuration document metadata schema
+
+## Synopsis
+
+Defines a set of informational key-value pairs for the configuration.
+
+## Metadata
+
+```yaml
+SchemaDialect: https://json-schema.org/draft/2020-12/schema
+SchemaID:      https://raw.githubusercontent.com/PowerShell/DSC/main/schemas/2023/10/config/document.metadata.json
+Type:          object
+```
+
+## Description
+
+Defines a set of informational key-value pairs for the configuration. Except for the
+`Microsoft.DSC` property, this metadata isn't validated. You can pass any data into your configuration as a property of `metadata`.
```

Comment:
Should we have stronger language that `Microsoft.DSC` is reserved to only be used by DSC itself?